avocado/core/extension_manager.py

```diff
@@ -32,6 +32,24 @@
 LOG_UI = logging.getLogger("avocado.app")
 
 
+PLUGIN_PRIORITY_VERY_HIGH = 100
+PLUGIN_PRIORITY_HIGH = 70
+PLUGIN_PRIORITY_NORMAL = 50
+PLUGIN_PRIORITY_LOW = 30
+PLUGIN_PRIORITY_VERY_LOW = 1
+
+
+def sort_extensions(extension):
```

Comment:
Looks like this method is really acting on the `Extension` or `ExtensionManager` object.  I would recommend to move this to a class method inside the Extension or ExtensionManager for keeping the responsibility aspect of those classes, something like:

```python
Extension.sort_by_priority()
```

Reply:
sure, make sense